### ISSUE TITLE
[CNN] 흐앙 ㅠㅠㅠ 첫 train data 돌리기 성공 ㅠㅠㅠ 얘들아 우린 할수있다 

### ISSUE THREAD

**Issue by lini1634** [PARKMIHE/AICOCO#25]
데이터 가공 참고함 http://www.birc.co.kr/2018/02/26/%EC%8B%A4%EC%A0%9C-%EC%9D%B4%EB%AF%B8%EC%A7%80-%EB%8D%B0%EC%9D%B4%ED%84%B0%EB%A5%BC-%ED%99%9C%EC%9A%A9%ED%95%9C-cnn-%EB%AA%A8%EB%8D%B8-%EA%B5%AC%ED%98%84%ED%95%98%EA%B8%B0/

저 위에꺼 차근차근 읽어보면 라벨링하고 다 해줌 ㅇㅇㅇㅇ

그리고 인공지능 수업때 선형분류 할때 앞에 데이터 로드하는부분 참고하고
CNN퀴즈 답 교수님께서 올려주신고 참고함 !!

![image](https://user-images.githubusercontent.com/44723287/58719263-2d88f300-840a-11e9-8dfb-ef69257c54ca.png)
로컬 런타임에 연결하기 복잡해서 로컬 설정하면서 구글 드라이브로 데이터도 올리고 있었는데 구글드라이브로 데이터가 더 빨리 올라가서 그냥 구글 드라이브로 했어!!

![image](https://user-images.githubusercontent.com/44723287/58719358-64f79f80-840a-11e9-85c6-76a4967342c5.png)

요거는 제일 위에 데이터 가공 참고한 곳에서 다 가져왔어 !! 주소만 내껄루 했다 
![image](https://user-images.githubusercontent.com/44723287/58719415-8bb5d600-840a-11e9-9202-88fbafa00454.png)

이거는 다들 익숙한 코드지 ?/ ㅋㅋ 밑에 세션 열어주고 변수초기화 해주는 부분 잘림 

![image](https://user-images.githubusercontent.com/44723287/58719442-a1c39680-840a-11e9-8f57-34ee0abcbcc1.png)

학습고고링 합니다이 
요것도 다들 익숙한 code~

![image](https://user-images.githubusercontent.com/44723287/58719472-b869ed80-840a-11e9-86b8-3bed168f0bfc.png)

쨔쟌~~~ 처음 돌렸는데 cost 저정도면 씹상타취 인정?

데이터 처음에 폴더 갯수떄문인지 뭔지 몇개 잘못저장되더니 10000개 쫌 넘었음 
저거 쫌 정리하고 learning_rate, batch_size, epoch조절하면서 CNN으로 성능 뽑을 만큼 뽑아보쟝

우리 이제 해야할꺼 -> traindata세미가 가공해준거 왕창 쓰고, test데이타도 따로 만들기 !

**Comment by lini1634** [comment]
learning rate 0.00003 epoch 50 하니깐
![image](https://user-images.githubusercontent.com/44723287/58721161-2f08ea00-840f-11e9-8a82-a6c85f547b57.png)
이렇게 나온다!!
